shared/substrate/types.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const BridgePalletName = "ChainBridge"
-const BridgeStoragePrefix = "Bridge"
+const BridgeStoragePrefix = "ChainBridge"
```

Comment:
just curious to why we updated this now

Reply:
It was renamed in the pallet as well, was causing a funky bug as another pallet was using the same name.